components/renku_data_services/project/models.py

```diff
@@ -24,11 +26,12 @@ class BaseProject:
     creation_date: datetime = field(default_factory=lambda: datetime.now(UTC).replace(microsecond=0))
     updated_at: datetime | None = field(default=None)
     repositories: list[Repository] = field(default_factory=list)
-    description: Optional[str] = None
-    keywords: Optional[list[str]] = None
-    documentation: Optional[str] = None
-    template_id: Optional[ULID] = None
+    description: str | None = None
+    keywords: list[str] | None = None
+    documentation: str | None = None
+    template_id: ULID | None = None
     is_template: bool = False
+    secrets_mount_directory: PurePosixPath | None = None
```

Comment:
question: is there a reason why we dont make this equal by default to a `constants.py` to value?

```
secrets_mount_directory: PurePosixPath = contants.SOMETHING
```

Reply:
Oh, this will be nicer to have, thanks.

Reply:
I may also update the migration accordingly, to set the default value.

Reply:
Refactored in 78b6cf4bb2cb147aec796aa013f4b5e3e31da7ad. Note that it is too tedious to do it at the model.